modules/delivery/delivery.service.js

```diff
@@ -0,0 +1,209 @@
+const axios = require('axios')
+const { 
+  ORDER_CREATION_FAILED,
+ } = require('../../error-messages/delivery.message')
+
+class NovaPoshtaService {
+  async apiNovaPoshtaRequest(properties, model, method){
```

Comment:
Please, use the verb for the method name

Reply:
fixed